sia/core_test.go

```diff
@@ -26,14 +26,27 @@ func establishTestingEnvironment(t *testing.T) (c *Core) {
 	consensus.MaxAdjustmentUp = big.NewRat(1005, 1000)
 	consensus.MaxAdjustmentDown = big.NewRat(995, 1000)
 
+	// mr is used to resolve conflicts between packages and variable names
+
 	// Pull together the configuration for the Core.
 	state, _ := consensus.CreateGenesisState() // The missing piece is not an error. TODO: That missing piece is deprecated.
 	walletFilename := "test.wallet"
 	w, err := wallet.New(walletFilename)
 	if err != nil {
+		t.Fatal(err)
+	}
+	hdb, err := hostdb.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	mrHost, err := host.New(state, w)
+	if err != nil {
+		return
+	}
+	mrRenter, err := renter.New(state, hdb, w)
+	if err != nil {
```

Comment:
not a fan of this naming convention. Normally I would use one-letter names here, but that's a problem because you have both `hostdb` and `host`. Maybe use `HostDB, `Host`, and `Renter`.

Also, some of these call `t.Fatal` while others just `return`.